index.bs

```diff
@@ -0,0 +1,221 @@
+
+</div>
+
+Usage Example {#example}
+------------------------
+
+<pre class="example highlight">
+    var observer = new PerformanceObserver(function(list) {
+        var perfEntries = list.getEntries();
+        for (var i = 0; i < perfEntries.length; i++) {
+            // Process entries
+            // report back for analytics and monitoring
+            // ...
+        }
+    });
+
+    // register observer for layout shift notifications
+    observer.observe({entryTypes: ["layoutShift"]});
+</pre>
+
+Terminology {#sec-terminology}
+==============================
+An <a>element</a>’s <dfn export>starting point</dfn> refers to the <a>element</a>'s <a>flow-relative offset</a> in the document.
+
+The <dfn export>visual representation</dfn> of a <a href="https://www.w3.org/TR/html401/struct/global.html#h-7.5.3">block-level</a> <a>element</a> is its <a>border box</a>. The <a>visual representation</a> of an <a href="https://www.w3.org/TR/html401/struct/global.html#h-7.5.3">inline element</a> is the geometric union of its <a>box fragment</a>s, the first of which is used to determine its <a>starting point</a>. 
+
+
+
+
+{{LayoutShift}} interface {#sec-layout-shift}
+=======================================
+
+<pre class="idl">
+    interface LayoutShift : PerformanceEntry {
+      readonly attribute long value;
+    };
+</pre>
+
+{{LayoutShift}} extends the following attributes of {{PerformanceEntry}} interface:
+
+* The {{PerformanceEntry/name}} attribute must return the {{DOMString}} <code>"layout-shift"</code>
+* The {{PerformanceEntry/entryType}} attribute must return <code>"layoutShift"</code>.
+* The {{PerformanceEntry/startTime}} attribute must return a {{DOMHighResTimeStamp}} of when the shift occured.
+* The {{PerformanceEntry/duration}} attribute must return 0.
+* The <dfn attribute for=LayoutShift>value</dfn> attribute's getter will return the value it is set to.
+
+<dfn attribute for=PerformanceLongTaskTiming>attribution</dfn>
+A user agent implementing {{LayoutShift}} must perform the following steps:
+1. Run the <a>register a performance entry type</a> algorithm with <code>"layoustShift"</code> as input.
+
+
+NOTE: The <a>getEntriesByType</a> method will not return entries of this type.  To obtain instability events that occur before a <a>PerformanceObserver</a> has been registered, register the observer with <a>buffered</a> set to true.
+
+
+
+
+Processing model {#sec-processing-model}
+========================================
+
+
+Modifications to the HTML specification {#sec-modifications-HTML}
+--------------------------------------------------------
+
+<em>This section will be removed once the <a href=https://html.spec.whatwg.org/multipage>HTML specification</a> has been modified.</em>
```

Comment:
Should this be a note? I'm actually not sure, but it feels strange as normative text.

Reply:
I was going with consistency with the event timing spec: https://wicg.github.io/event-timing/#sec-events-exposed